src/third-party/pinyin/pinyin_data.hpp

```diff
@@ -233,10 +236,10 @@ static const std::map<std::u32string, std::u32string> pinyin_data = {
     {U"min", U"㞴㞶㟩㟭㢯㥸㨉㬆㳷䁕䂥䃉䋋䝧䟨䡑䡻䪸䲄僶冺刡勄厸呡垊姄岷崏忞怋悯惽愍慜憫抿捪敃敏敯旻旼暋民汶泯渂湏湣潣玟珉琘琝瑉痻皿盷盿眠砇碈笢笽簢緍緡繩缗罠苠蠠鈱錉鍲閔閩闵闽鰵鳘鴖黽𠊟𢼖𢽹𣱈𣱉𣷠𣹒𤇜𤛎𤸅𤺖𤿕𥜐𦈏𦌡𦫮𦳜𧁋𧌙𧲃𧵣𨏵𩔉𩭷𪂆𪄴𪉎"},
     {U"ming", U"㝠㟰㫥䄙䆩䊅䏃䒌䫤䳟佲冥凕名命姳嫇慏掵明暝朙椧榠洺溟猽皿眀眳瞑茗萌蓂螟覭詺鄍酩銘铭鳴鸣𠋶𡥸𥌏𥥊𥹆𥿨𦡉𦫭𧟠𧱴𩣶𪂤𪗸"},
     {U"miu", U"嘐繆謬谬𨱯"},
+    {U"m", U"呣"},
```

Comment:
Still needs to be merged with line 222

Reply:
Thank you again for catching this, I did not expected it to happen